examples/worlds/actors_population.sdf.erb

```diff
@@ -138,13 +142,26 @@
     </include>
 
     <%
-      # number of population
```

Comment:
The dude on line 141 seems a bit lost in the crowd, is he needed?

Reply:
Do you mean this `# number of population` line? I think you are looking at the old file? This is the new change equivalent to that line https://github.com/ignitionrobotics/ign-gazebo/pull/336/files#diff-b1617932cd58db140bca53385d0da9bcR145

Reply:
Oh sorry, I meant the included actor around line 141 of the new file: 

```
    <include>
      <uri>https://fuel.ignitionrobotics.org/1.0/Mingfei/models/actor</uri>
    </include>
```

Reply:
I remember the lonely actor used to be able to walk around others without bumping into anyone :)

Reply:
Ooo that's what it was. I was wondering why there's always an extra actor different from others. Yeah I'll remove it.